README.development

```diff
@@ -76,3 +76,24 @@ You can use homebrew to install some dependencies:
 $ brew install python mpv lame portaudio protobuf npm rustup-init gettext rename
 
 $ brew link gettext --force
+
+Windows users (using Visual Studio)
+----------
+
+1. Download and install Cygwin and put its `/bin/` directory on your system path.
+1. Install the Cygwin Packages: `apt-cyg install rsync make`
+    1. If the Cygwin `/usr/bin/` directory exists, move all files from `/usr/bin/` to `/bin/`.
+    1. Download `gettext` 0.20.1 or superior and put its `bin` directory on your system path.
+        1. https://mlocati.github.io/articles/gettext-iconv-windows.html
+1. Download and install Python for Windows (not from Cygwin) and put `python.exe` (not `python3.exe`) on your system path.
+1. Download and install pip for your Windows Python.
+1. Download and install rust (compiler), npm, git and put them your system path.
+1. Download and install Visual Studio 2017 or superior and:
+    1. `git clone https://github.com/evandroforks/portaudio`
```

Comment:
There's no need for that, people can download binary wheels:

https://www.lfd.uci.edu/~gohlke/pythonlibs/#pyaudio

Reply:
I tried using those wheels, but they fail because they do not contain the portaudio library embed on the file. If you notice they file size, they have only ~100KB, while portaudio.lib (statically linked) has about 2MB.

Even with the portaudio.lib  file, just using those wheels is not enough because I also had to modify the `setup.py` to pass the flag: `/DEFAULTLIB:advapi32.lib`, otherwise the linker will point out 2 missing symbols.

Reply:
I think they should work - they are what I use to build the Windows version of Anki.

Reply:
> I tried using those wheels, but they fail because they do not contain the portaudio library embed on the file

Those wheels have the portaudio library statically linked. They certainly work.

Reply:
I see. May be I did something wrong or my Computer has something not compatible with those wheels libraries. I already did all the changes with the latest push.

Instead of removing my instructions, updated it to use both, then, someone else like me which also had trouble using those wheels, they can have a reliable option to use.

Reply:
I don't think most users will need that. How about putting those instructions in the readme at the top of your forked repo, and just pointing users to that readme if the wheels don't work for them?

Reply:
I moved the instructions to my pyaudio fork. This why is better because the instructions to install the pyaudio should be in the pyaudio repository.